.drone/lcov-report.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+SKIPLIST="math"
+
+: "${LCOV_SKIP_PATTERN:='^[9]'}" # Set default lcov skip pattern
+
+pwd
+
+export CODECOV_SCRIPT=${BOOST_CI_SRC_FOLDER}/ci/travis/codecov.sh
+export CI_DIR=${BOOST_CI_SRC_FOLDER}/ci
+export BOOST_CI_CODECOV_IO_UPLOAD="skip"
+export LCOV_VERSION="v2.3"
+export LCOV_IGNORE_ERRORS_LEVEL=standard
+export EXPORT_BOOST_SRC_DIR="yes"
```

Comment:
Isn't this unused?

Reply:
Partially unused.  Ok, I have removed BOOST_CI_CODECOV_IO_UPLOAD, since "collect" doesn't need that, and EXPORT_BOOST_SRC_DIR was a remnant of another script.
What's left are LCOV_VERSION and LCOV_IGNORE_ERRORS_LEVEL.    These match `codecov.sh` currently.  But reports could easily need those variables to be modified, in the future, testing a newer lcov.